src/foremast/securitygroup/create_securitygroup.py

```diff
@@ -311,3 +281,48 @@ def create_security_group(self):  # noqa
 
         self.log.info('Successfully created %s security group', self.app_name)
         return True
+
+    def create_ingress_rule(self, app, rule):
+        """Create a normalized ingress rule.
+
+        Args:
+            app (str): Application name
+            rule (dict or int): Allowed Security Group ports and protocols.
+
+        Returns:
+            dict: Contains app, start_port, end_port, protocol, cross_account_env and cross_account_vpc_id
+
+        """
+        if isinstance(rule, dict):
+            # Advanced
+            start_port = rule.get('start_port', 80)
```

Comment:
I don't like auto-assuming connectivity in terms of ports. This could accidentally expose port 80 in the event of misconfigured files.

Reply:
Good point. Fixed.